test_suite/generic/test_generic.py

```diff
@@ -173,6 +173,18 @@ def test_release_version(self, host, instance_data):
             f'product version ({product_version}) does not match package release version'
         assert product_version.replace('.', '-') in cloud_image_name, 'product version is not in image name'
 
+    def test_sshd(self, host):
+        """
+        sshd service shoud be on, password authentication shoud be disabled
+        """
+        with host.sudo():
+            sshd = host.service('sshd')
+            assert sshd.is_running, 'ssh.service is not active'
+
+            if not host.file('/etc/ssh/sshd_config').contains('PasswordAuthentication no') and \
+               not host.file('/etc/ssh/sshd_config').contains('#PasswordAuthentication yes'):
```

Comment:
We could be pickier in this test.

Check what is being done here:
https://github.com/osbuild/cloud-image-val/blob/4944549b6a824d32c2e4e992f03b7e096df2422d/test_suite/cloud/test_aws.py#L220